### DISCUSSION TITLE
Support compressed rosbags (*.mcap.zstd)

### DISCUSSION THREAD

**Discussion by bittsitt** [foxglove/community#878] [Q&A] [answered]
It is not possible to open a compressed mcap-rosbag in foxglove studio. mcap.zstd is not listed in file filter, and selecting all files and trying to open it that way doesn't seem to work.

**Comment by defunctzombie** [comment, marked as answer]
You can open an mcap file that uses `zstd` chunk compression (a feature of mcap) not an mcap that is itself entirely compressed. We do not recommend compressing the entire mcap file and instead using the chunk compression feature. If you compress the entire mcap file it must be fully decompressed to read any part of it while chunk compression allows for reading parts like the summary sections and message indexes without decompressing the chunks unless needed.
